clarify signed blob transaction hashing (#6238)
ethereum/EIPs: EIPS/eip-4844.md

    return ecrecover(tx_hash(tx), int(sig.y_parity)+27, sig.r, sig.s)
```

> The transaction hash of a signed blob transaction should be computed per [`EIP-2718`](./eip-2718.md):

Reviewer: EIP-2718 doesn't specify how transaction hashes are calculated.  I *believe* that transaction hashes are the `keccak256` of the bytes in the transaction trie whose root is in the block header, so if the below aligns with that then it is correct.
```suggestion
The transaction hash of a signed blob transaction should be computed as:
```

```python
def signed_tx_hash(tx: SignedBlobTransaction) -> Bytes32:
    return keccak256(BLOB_TX_TYPE + ssz.serialize(tx))

> ```	

Reviewer: ````suggestion
```
````
